lib/streamlit/delta_generator.py

```diff
@@ -117,12 +116,12 @@ class DeltaGenerator(
 
     Parameters
     ----------
-    container: BlockPath_pb2.BlockPath or None
+    container: BlockPath_pb2.BlockPath.ContainerValue or None
```

Comment:
Can this be just `BlockPath.ContainerValue`?

Reply:
I think that would be the wrong type annotation. But: this points to a broader question - for our internal APIs at least, do we need to continue documenting parameter types in comments if we're also documenting their types with annotations? (I'd vote for stopping this)